### ISSUE TITLE
Support textures

### ISSUE THREAD

**Issue by onlyuser** [syoyo/MMDLoader#17]

**Comment by onlyuser** [comment]
Low hanging fruit.

**Comment by syoyo** [comment]
Texture in MMD has somewhat tricky. At least you need also support corresponding (GLSL) shader for correct texture mapping.

**Comment by onlyuser** [comment]
Yeah, I see uv coordinates for vertex, but no reference to material for bone/face/mesh.

Not as straight-forward as I had expected.
